README.md

```diff
@@ -18,7 +18,9 @@ This is now done by Github Actions automatically, on push to main branch or when
 
 ## Terraform Usage
 
-To run the Terraform configuration, follow these steps:
+** Note that terraform configuration in network directory will setup initial VPC and subnets, only runs once by administrator.
+To run the Terraform configuration as a user, follow these steps in user directory under terraform:
+As a pre-requisite step, create a key-pair in AWS EC2 console, and place the key in user directory.
```

Comment:
Might want to list the directory here as the users might not have a lot of experience with AWS.

Reply:
Addressed.